metapool/__init__.py

```diff
@@ -19,9 +21,11 @@
 from .amplipool import assign_emp_index
 from .igm import IGMManifest
 from .count import run_counts
+from .controls import is_blank, get_blank_root
 from .metapool import (extract_stats_metadata, sum_lanes, compress_plates,
                        add_controls, generate_override_cycles_value,
                        TUBECODE_KEY)
+from .controls import is_blank
```

Comment:
it looks like is_blank is being imported twice?

Reply:
It looks like it. I recommend setting up a standard linter, like ruff, which is easy to run locally in dev environments as well as in CI

Reply:
It might be possible to configure flake8 to check for this type of issue as well. Adding ruff or pylint to our CI scripts would probably cover a larger set of errors however.

Reply:
Yes, I failed to run flake8 *one last time* ... when I did, this is the one and only thing it found :)

Reply:
`ruff` is fast enough that it can run continuously, and it's likely the editor being used probably already has very easy support for it

Reply:
FWIW I use ruff and pyright in Helix editor, and I believe it's installed as part of the vscode extensions for python. It is available as a plugin for pycharm but I don't use it in that case.